reports-server/main.py

```diff
@@ -0,0 +1,26 @@
+from http.server import ThreadingHTTPServer
```

Comment:
On the [documentation](https://docs.python.org/3/library/http.server.html), it says:
> Warning: http.server is not recommended for production. It only implements basic security checks.

Have you taken that warning into account?

Reply:
The server itself will not be exposed to the internet, it will be behind a reverse proxy (I will setup `nginx`)